boards/arm/nucleo_wl55jc/nucleo_wl55jc.yaml

```diff
@@ -0,0 +1,14 @@
+identifier: nucleo_wl55jc
+name: ST Nucleo WL55JC
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 64
+flash: 256
+supported:
+  - gpio
+  - counter
```

Comment:
From what I can see counter is not yet supported

Reply:
Right. Fixed.